src/components/Choropleth.vue

```diff
@@ -129,29 +138,46 @@ const style = (f: Feature) => {
 };
 
 const updateBounds = async () => {
-    // update bounds
+    const rawMap = toRaw(map.value);
+    if (!rawMap) return;
+    const leafletMap = rawMap.leafletObject;
+
     const country = selectedCountryId.value || "GLOBAL";
     const [west, east, south, north] = countryBoundingBoxes.value[country];
     const countryBounds = new LatLngBounds([south, west], [north, east]);
-    await map.value.leafletObject.fitBounds(countryBounds);
+    await leafletMap.fitBounds(countryBounds);
 
     // record bounds for testing
     bounds.value = { west, east, south, north };
 };
 
 const updateMap = async (newFeatures: Feature[]) => {
-    if (!map.value) return;
+    const rawMap = toRaw(map.value);
+    if (!rawMap) return;
+    const leafletMap = rawMap.leafletObject;
 
```

Comment:
These three lines recur twice, could they go into a `computed`?

Reply:
unfortunately the map is already a ref in vue, I am unpacking vue reactivity here, so i can't move it into a computed because that just wraps in it vue reactivity again. I can however do something like
```typescript
const leafletMap = toRaw(map.value)?.leafletObject;
if (!leafletMap) return;
```
to reduce the line count :laughing:

Reply:
(I'm more concerned with avoiding duplication than with reducing line count)

Reply:
ohhh i got you, yes i can extract this into a function!